### PR TITLE
use existing alias when emitting symbols

### PR DESCRIPTION
**Pull request by martinklepsch**
WIP to address https://github.com/squint-cljs/squint/issues/472

Please answer the following questions and leave the below in as part of your PR.

- [x] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).
- [x] This PR contains a [test](https://github.com/babashka/babashka/blob/master/doc/dev.md#tests) to prevent against future regressions
- [x] I have updated the [CHANGELOG.md](https://github.com/squint-cljs/squint/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

File: src/squint/compiler_common.cljc [hunk 1]
```diff
@@ -248,9 +248,9 @@
                          (when (= "js" sym-ns)
                            (munge* (name expr)))
                          (when-let [resolved-ns (get @*aliases* (symbol sym-ns))]
-                           (str (if (symbol? resolved-ns)
-                                  (munge resolved-ns)
-                                  sym-ns) "." #_#_sym-ns "_" (munged-name sn)))
+                           (str sym-ns "."
+                                #_#_sym-ns "_"
```
Comment (martinklepsch) [inline src/squint/compiler_common.cljc:252]: could this be deleted?

**Comment by martinklepsch** [comment]
@borkdude would you mind pointing me in the right direction for where to add tests? I found a few different test setups 😅

File: src/squint/compiler_common.cljc (hunk 1 above)
Comment (borkdude) [reply src/squint/compiler_common.cljc:252]: probably

**Comment by borkdude** [comment]
Merged manually, thanks!

**Pull request closed without merging**